odinmain/src/main/kotlin/me/odinmain/features/impl/dungeon/BloodCamp.kt

```diff
@@ -177,15 +178,15 @@ object BloodCamp : Module(
 
         val ping = ServerUtils.averagePing
```

Comment:
why?

Reply:
soopy had stuff for ping so i carried that on.

Reply:
sure, but it doesnt need to be a seperate variable, just use ServerUtils.averagePing or import averagePing from it